paypro-gateways-woocommerce/includes/paypro/wc/gateway.php

```diff
@@ -1,18 +1,15 @@
 <?php if(!defined('ABSPATH')) exit; // Exit if accessed directly
 
-abstract class PayPro_WC_Gateway_Abstract extends WC_Payment_Gateway
+class PayPro_WC_Gateway extends WC_Payment_Gateway
 {
-    protected $default_title;
-
-    protected $default_description;
-
-    protected $default_logo;
-
     protected $issuer;
+    protected $gateway_title = '';
```

Comment:
We could also make these constants which can be changed by the child class.

Reply:
`issuer`, `gateway_title` and `gateway_description`?